index.html

```diff
@@ -418,6 +468,21 @@
 [Deployment](https://developer.apple.com/videos/play/wwdc2025/232/) [is](https://developer.chrome.com/blog/digital-credentials-api-origin-trial).
 
 
+### Authorizing Use {#auth-use}
+
+Many uses of digital credentials will rely on asking people to make a decision
+about whether to authorize the use of the credential.
+
+It is challenging to design a system
+that can convey the details
+of what people are being asked to approve given the potential diversity of uses
+in a sufficiently comprehensible manner.
+User agents need to help their users understand what information is disclosed
+to enable an informed choice.
+This is particularly important for non-obvious data or metadata,
+that make some of the finer distinctions obvious.
```

Comment:
I think we lost something here: how is the "non-obvious data" making the finer distinctions obvious? What finer distinctions?

Reply:
Fair.  That sentence was a bit opaque in retrospect.  I've reworded the paragraph.